docs/quick/k8s/submarine-kubeflow.md

```diff
@@ -0,0 +1,112 @@
+```
+kind create cluster --image kindest/node:v1.15.6 --name submarine
+```
+
+> NOTICE: Kuebflow 0.7 compatible with kubernetes 1.14 and 1.15
+
+### Deploy Dashboard
+To deploy Dashboard, execute following command:
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+```
+
+### Create RBAC
+```
+kubectl create serviceaccount cluster-admin-dashboard-sa
+kubectl create clusterrolebinding dashboard-admin-sa --clusterrole=cluster-admin --serviceaccount=default:dashboard-admin-sa
+```
+
+### Get access token
+```
+kubectl get secrets
+kubectl describe secret dashboard-admin-sa-token-6nhkx
+```
+
+### Running
+Run the following command:
+```
+kubectl proxy
+```
+
+Now access Dashboard at:
+> http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/
+
+## Prepare Kubeflow
+
+### Download kfctl
+Download the kfctl v0.7.0 release from the [Kubeflow Releases Page](https://github.com/kubeflow/kubeflow/releases/tag/v0.7.0). More info see [Kubeflow Prepare your environment](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/#prepare-your-environment).
+
+Run the following command:
+```
+tar -xvf kfctl_v0.7.0_darwin.tar.gz
+mv kfctl_darwin /usr/local/bin/kfctl
+```
+
+### Deploy Kubeflow
+We customlize the kubeflow compnents based the [offical manifests](https://raw.githubusercontent.com/kubeflow/manifests/v0.7-branch/kfdef/kfctl_k8s_istio.0.7.1.yaml), and output into `example/kfctl_k8s_istio.0.7.1.yaml`.
+
+Run the following command to deploy the kubeflow:
+```
+kfctl apply -V -f example/kfctl_k8s_istio.0.7.1.yaml
+```
+
+Check the resources deployed in namespace kubeflow:
+```
+kubectl -n kubeflow get all
+```
+
+## Prepare Submarine Server
+
+### Get package
+You can dowload submarine 0.3.0 from releases page or build from source.
+
+### Configuration
+Copy the kube config into `conf/k8s/config` or modify the `conf/submarine-site.xml`:
+```
+<property>
+  <name>submarine.k8s.kube.config</name>
+  <value>PATH_TO_KUBE_CONFIG</value>
+</property>
+```
+
+### Start Submarine Server
+```
+./bin/submarine-daemon.sh start getMysqlJar
+```
+
+> http://127.0.0.1:8080/api/v1/jobs
+
+## Submit Submarine Job by REST
+You can use the Postman post the job to server or use curl run following command:
+```
+curl -H "Content-Type: application/json" --request POST \
+--data `{"name":"mnist","librarySpec":{"name":"TensorFlow","version":"2.1.0","image":"gcr.io/kubeflow-ci/tf-mnist-with-summaries:1.0","cmd":"python /var/tf_mnist/mnist_with_summaries.py --log_dir=/train/log --learning_rate=0.01 --batch_size=150","envVars":{"ENV_1":"ENV1"}},"submitterSpec":{"type":"k8s","configPath":null,"namespace":"submarine","kind":"TFJob","apiVersion":"kubeflow.org/v1"},"taskSpecs":{"Ps":{"name":"tensorflow","replicas":2,"resources":"cpu=4,memory=2048M,nvidia.com/gpu=1"},"Worker":{"name":"tensorflow","replicas":2,"resources":"cpu=4,memory=2048M,nvidia.com/gpu=1"}}}` \
```

Comment:
I suggest to create a job spec file. This is a key point we want to high-light for the user.